src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -373,6 +373,13 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="_CheckForUnnecessaryWindowsDesktopSDK"
         BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
         Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '5.0')) and '$(_MicrosoftWindowsDesktopSdkImported)' == 'true'">
-    <NETSdkWarning ResourceName="UnnecessaryWindowsDesktopSDK" />
+    <ItemGroup Condition="'$(TargetFrameworks)' != ''">
+      <_TargetFrameworks Include="$(TargetFrameworks.Split(';'))" />
+      <_TargetFrameworksBelowNet5 Include="@(_TargetFrameworks)" />
+      <_TargetFrameworksBelowNet5 Remove="@(_TargetFrameworks)" Condition="'$([MSBuild]::GetTargetFrameworkIdentifier(%(Identity)))' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($([MSBuild]::GetTargetFrameworkVersion(%(Identity), 2)), '5.0'))" />
```

Comment:
This type of parsing doesn't respect possible TargetFramework aliasing.  If we want to prevent the warning, let's just prevent it for all multitargeted projects.